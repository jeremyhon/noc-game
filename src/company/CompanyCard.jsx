import React from 'react'
import PropTypes from 'prop-types'

import Card from 'components/card'
import skillsPropType from 'components/stats/skillsPropType'

const CompanyCard = (props) => {
  return (
    <Card
      className="company-card"
      title={props.name}
      subtitle="Needs"
      skills={props.needs}
    />
  )
}

CompanyCard.propTypes = {
  name: PropTypes.string.isRequired,
  needs: skillsPropType.isRequired,
}

export default CompanyCard
