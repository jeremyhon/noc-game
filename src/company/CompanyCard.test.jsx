import React from 'react'
import { shallow } from 'enzyme'

import CompanyCard from './CompanyCard'

it("matches the snapshot", () => {
  const props = {
    name: "EVE ONLINE",
    needs: {
      business: 1,
      technical: 1,
      design: 1,
    }
  }
  expect(shallow(<CompanyCard {...props} />)).toMatchSnapshot()
})
