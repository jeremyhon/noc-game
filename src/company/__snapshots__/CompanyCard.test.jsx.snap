// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<Card
  className="company-card"
  skills={
    Object {
      "business": 1,
      "design": 1,
      "technical": 1,
    }
  }
  subtitle="Needs"
  title="EVE ONLINE"
/>
`;
